src/Pdk/WcPdkBootstrapper.php

```diff
@@ -219,17 +219,17 @@ protected function getAdditionalConfig(
             'routeBackendPdk'                => value('pdk'),
             'routeBackendWebhook'            => value('webhook/(?P<hash>.+)'),
             'routeBackendPermissionCallback' => factory(static function (): string {
-                $callback = '__return_false';
-
-                if (is_user_logged_in() && 'shop_manager' === (wp_get_current_user()->roles[0] ?? '')) {
-                    $callback = '__return_true';
-                }
-
-                if (current_user_can('manage_options')) {
-                    $callback = '__return_true';
+                if (! is_user_logged_in()) {
+                    return '__return_false';
                 }
                 
-                return $callback;
+                foreach (wp_get_current_user()->roles as $role) {
+                    if (in_array($role, ['shop_manager', 'administrator'])) {
```

Comment:
Is het de bedoeling dat `if (current_user_can('manage_options'))` er niet meer in zit?

Reply:
'manage_options' is een capability die volgens mij alleen de 'administrator' heeft. Om het 'simpel' te houden testen we nu alleen op rollen, en niet op capabilities. Ik denk dat het in de praktijk logischer werkt ook. 'manage_options' klopt eigenlijk natuurlijk alleen vwb de settings schermen, en niet vwb het exporteren van zendingen, want dat laatste is zeker niet het managen van opties.